base/applications/regedit/childwnd.c

```diff
@@ -129,84 +128,6 @@ static void finish_splitbar(HWND hWnd, int x)
     ReleaseCapture();
 }
 
-/*******************************************************************************
- *
- *  Key suggestion
- */
-
-#define MIN(a,b)    ((a < b) ? (a) : (b))
-
-static void SuggestKeys(HKEY hRootKey, LPCWSTR pszKeyPath, LPWSTR pszSuggestions,
```

Comment:
Where was this "keys suggestion" feature used? (do you have a quick screenshot?)

Reply:
They were intended to be added to be added to the right-click-menu, see the code in the bottom of childwnd and framewnd. Have never seen this been working in a contributional way. Therefore stripped.

Reply:
https://github.com/reactos/reactos/commit/32cf34124f5277185970b7aaf20055f03f1f1fa8

Reply:
It's not worth the additional complexity, and Windows doesn't have it. I checked on XPSP3.

Reply:
Windows does not have it is NO reason to remove a useful feature IMO. We are NOT cloning a OS here, we are improving, too.